src/sign.proj

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project="dir.props" />
+  <Import Project="..\dir.targets" />
+
+  <Target Name="Build"
+          Condition="'$(SkipSigning)' != 'true' and '$(SignType)' != 'public'">
+
+    <ItemGroup>
+      <SignBuildArgs Include="/p:ArchGroup=$(ArchGroup)" />
```

Comment:
What did you do to figure out the set of properties to pass?

Reply:
A combination of things:

1. I looked at the properties that were being passed on the command line during the official build: https://github.com/dotnet/corefx/blob/d8ae2adab78bc5b4088a18d8cf826867d39774b3/buildpipeline/DotNet-CoreFx-Trusted-Windows.json#L185
2. I ran through all the used properties in `src/sign.builds` and looked at how they were calculated - mostly this was just `$(NativeBinDir)`, and ensured the right properties were flown.